cms.protractor.helper.js

```diff
@@ -28,6 +28,13 @@ CMS.Protractor = {
         }, this.mainElementsWaitTime);
     },
 
+    // utility to test expectation an element is missing
+    expectByCssToBeAbsent: function (element) {
```

Comment:
Then `expectToBeAbsent` should fit - it's short and descriptive + we also have `waitFor` and `waitForDisplayed` without `Element` - light and short